docs/pages/older-versions.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Older Versions of Teleport Documentation
+description: Links to older versions of the Teleport Docs
+h1: GitHub links to older versions of the Teleport Docs
+layout: tocless-doc
+---
+## Older Versions of Teleport Docs
+
+Deprecated versions of the Teleport docs can be found at the GitHub links below:
+
+[Teleport 6.2](https://github.com/gravitational/teleport/tree/branch/v6.2) <br/>
```

Comment:
Would this list look nicer as a `TileSet`/`Tile`s?

Reply:
You're welcome to style it however you like, for now this is just a placeholder for testing purposes.